src/libs/HttpUtils.js

```diff
@@ -109,7 +118,12 @@ function xhr(command, data, type = CONST.NETWORK.METHOD.POST, shouldUseSecure =
     });
 
     const url = ApiUtils.getCommandURL({shouldUseSecure, command});
-    return processHTTPRequest(url, type, formData, data.canCancel);
+    return processHTTPRequest(url, type, formData, data.canCancel, command);
+}
+
+function cancelPendingReconnectAppRequests() {
```

Comment:
There's only one request hooked to this controller at any given time, is that correct? Or we can have multiple `ReconnectApp` requests?

Reply:
Yes, just only one ReconnectApp request appear after we are online again